library/core/src/lib.rs

```diff
@@ -354,3 +355,5 @@ pub mod arch {
         /* compiler built-in */
     }
 }
+
+include!(concat!("../../std/src/primitive_docs.rs"));
```

Comment:
nit: unnecessary `concat!`:

```suggestion
include!("../../std/src/primitive_docs.rs");
```

Reply:
Thanks, fixed.